webapp/publisher/snaps/views.py

```diff
@@ -1308,3 +1309,53 @@ def post_preview(snap_name):
     context["normalized_os"] = preview_data.get_normalised_oses()
 
     return flask.render_template("store/snap-details.html", **context)
+
+
+@publisher_snaps.route("/<snap_name>/builds", methods=["GET"])
+@login_required
+def get_snap_builds(snap_name):
+    try:
+        details = api.get_snap_info(snap_name, flask.session)
+    except ApiResponseErrorList as api_response_error_list:
+        if api_response_error_list.status_code == 404:
+            return flask.abort(404, "No snap named {}".format(snap_name))
+        else:
+            return _handle_error_list(api_response_error_list.errors)
+    except ApiError as api_error:
+        return _handle_errors(api_error)
+
+    context = {
+        "snap_id": details["snap_id"],
+        "snap_name": details["snap_name"],
+        "snap_title": details["title"],
+    }
+
+    snap = launchpad.snaps.findByStoreName(
```

Comment:
I find it this `snap` name a bit confusing. Usually our snap data comes from store API, but this one comes from LP, so is not the same as store snap. Also, it seems that it's a list rather than single snap (even if we are interested in just the first one).

Could we call this `lp_snaps` or something like that (to suggest it's a list and where it comes from)?